client/app/api/course/Video/Topics.js

```diff
@@ -0,0 +1,40 @@
+import { getVideoId } from 'lib/helpers/url-helpers';
+
+import BaseCourseAPI from '../Base';
+
+export default class TopicsAPI extends BaseCourseAPI {
+
+  /**
+   * Creates a video discussion post
+   *
+   * @param {object} fields
+   *   - params in the format of { :timestamp, :discussion_topic } }
+   * @return {Promise} A promise for the server's response.
+   */
+  create(fields) {
+    return this.getClient().post(this._getUrlPrefix(), fields);
+  }
+
+  /**
+   * Retrieves a video discussion topic
+   *
+   * @param {number} topicId The id of the topic to retrieve
+   * @return {Promise} A promise for the server's response.
```

Comment:
For APIs, we will document the format of the response from the server. You can use Survey APIs as a reference